decidim-core/app/cells/decidim/footer_topics_cell.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  # @todo Document this cell
+  #
+  # Example:
+  #
+  #    cell("decidim/footer_topics")
+  class FooterTopicsCell < Decidim::ViewModel
+    def show
+      return if topics.blank?
+
+      topics
```

Comment:
This should be `render`

```suggestion
      render
```